skytap/resource_skytap_environment.go

```diff
@@ -49,7 +49,7 @@ func resourceSkytapEnvironment() *schema.Resource {
 			"routable": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Default:  false,
+				Default:  nil,
```

Comment:
not sure about the reason for this change but it seems problematic, in the current schema the change.

The creation will set a default value, this default value will not match this null value so it will probably be reflected as change.

Reply:
The reason for this change is because the routable attribute of an environment created from an empty template (no network nor environments) is nil. This wasn't considered originally. In hindsight it would have probably been better to leave the schema as it was and to have worked around the nil `routable` attribute. I will subsequently do the latter...